gulpfile.js

```diff
@@ -206,7 +206,7 @@ function endBuildStep(stepName, targetName, startTime) {
   const endTime = Date.now();
   const executionTime = new Date(endTime - startTime);
   const secs = executionTime.getSeconds();
-  const ms = executionTime.getMilliseconds().toString().padStart(3, '0');
+  const ms = ('000' + executionTime.getMilliseconds().toString()).slice(-3);
```

Comment:
These are not equivalent... If ms is more than 999 we'll get some weird values.

Reply:
@jridgewell `getMilliseconds()` returns a value between 0 and 999.